src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
```

Comment:
Of course, I always say you can't remove duplication until you produce it. :-)

Maybe we should merge it this way and look at simplifying later, particularly if the simplification spans more constraints and operators than the ones in play here.